### PR TITLE
relnote(117): Measuring Tool can be controlled using the keyboard

### PR DESCRIPTION
**Pull request by bsmth**
Selected area of the page can be moved using keyboard shortcuts

__Related issues and pull requests:__
- [x] Parent issue https://github.com/mdn/content/issues/28754

__Bugzilla:__
- https://bugzilla.mozilla.org/show_bug.cgi?id=1262782

### CODE REVIEW

**Comment by vinnydiehl** [comment]
Thanks for taking care of the changelog entry for this!

Perhaps we should mention that the arrow keys without any modifier will move the measuring tool?

**Comment by bsmth** [comment]
No problem!

> the arrow keys without any modifier will move the measuring tool?

Very good, I didn't realize - I've updated in https://github.com/mdn/content/pull/28764/commits/306626fcb561c9c6707264eb42b1a058a2bb58ba which should be accurate now, thank you for the hints.

File: files/en-us/mozilla/firefox/releases/117/index.md [hunk 1]
```diff
@@ -12,6 +12,10 @@ This article provides information about the changes in Firefox 117 that affect d
 
 ### Developer Tools
 
+- The area selected by the [Measuring Tool](https://firefox-source-docs.mozilla.org/devtools-user/measure_a_portion_of_the_page/index.html) can now be resized using the keyboard.
+  Pressing arrow keys moves the selected area, and <kbd>Ctrl</kbd> + arrow keys (<kbd>Cmd</kbd> + arrow keys on a Mac) resizes the selected area.
+  Holding <kbd>Shift</kbd> accelerates moving and resizing actions made with these key combinations ([Firefox bug 1262782](https://bugzil.la/1262782)).
```
Comment (dipikabh) [inline files/en-us/mozilla/firefox/releases/117/index.md:17]: minor reword suggestions

```suggestion
- The area selected by the [Measuring Tool](https://firefox-source-docs.mozilla.org/devtools-user/measure_a_portion_of_the_page/index.html) can now be resized and moved using keyboard shortcuts.
  Pressing the arrow keys moves the selected area, while pressing <kbd>Ctrl</kbd> + arrow keys (or <kbd>Cmd</kbd> + arrow keys on a Mac) resizes the selected area.
  Holding down the <kbd>Shift</kbd> key accelerates the moving and resizing actions when using these key combinations ([Firefox bug 1262782](https://bugzil.la/1262782)).
```

File: files/en-us/mozilla/firefox/releases/117/index.md [hunk 2]
```diff
@@ -12,6 +12,10 @@ This article provides information about the changes in Firefox 117 that affect d
 
 ### Developer Tools
```
Comment (dipikabh) [inline files/en-us/mozilla/firefox/releases/117/index.md:13]:
```suggestion
### Developer tools
```

**Review by dipikabh — APPROVED** [review]
I've left some minor edits, but this is ready to land.

The placement of "Developer tools" section has varied across releases. What do you think  about moving it to the end of the "## Changes for web developers" section or after "### JavaScript"? I feel HTML, CSS, and JavaScript should be on the top.

File: files/en-us/mozilla/firefox/releases/117/index.md (hunk 1 above)
Comment (bsmth) [reply files/en-us/mozilla/firefox/releases/117/index.md:17]: super tnx

**Comment by bsmth** [comment]
Thanks both! Merging shortly 👍🏻

**Pull request merged**